src/System.Runtime/tests/System/ActivatorTests.cs

```diff
@@ -92,10 +92,13 @@ public static void CreateInstance_Invalid()
             Assert.ThrowsAny<MissingMemberException>(() => Activator.CreateInstance(typeof(AbstractTypeWithDefaultCtor))); // Type is abstract
             Assert.ThrowsAny<MissingMemberException>(() => Activator.CreateInstance(typeof(IInterfaceType))); // Type is an interface
 
-#if netcoreapp
-            // Type is not a valid RuntimeType
-            AssertExtensions.Throws<ArgumentException>("type", () => Activator.CreateInstance(Helpers.NonRuntimeType()));
-#endif // netcoreapp
+#if netcoreapp || uapaot
```

Comment:
For these ifs, can you add a comment with a TODO for the issue number so we follow-up to remove them?